package.json

```diff
@@ -7,7 +7,9 @@
     "test": "test"
   },
   "scripts": {
-    "test": "./test/runtests.sh"
+    "test": "./test/runtests.sh",
+    "compile": "rm -rf lib/ && babel --source-maps -d lib/ src/",
```

Comment:
we should figure out a way to do that... i suspect the solution will include having to run a command that pulls things into the individual projects since I don't know of a way to share files into the root of another project with npm.  Maybe there is and I just haven't read it yet.